modules/registry-configuring-storage-gcp-user-infra.adoc

```diff
@@ -18,9 +18,33 @@ cloud credentials.
 contents of a credentials file provided by GCP:
 ** `REGISTRY_STORAGE_GCS_KEYFILE`
 
-
 .Procedure
 
-////
-Stub for procedure to manually set storage medium.
-////
+Use the following procedure if the Registry Operator cannot create a GCP bucket
+and automatically configure storage.
+
+. Set up a link:https://cloud.google.com/storage/docs/lifecycle[Lifecycle Management policy]
```

Comment:
The web page opens to a section named "Object Lifecycle Management" so the link metadata should reflect this to avoid any confusion.